app/views/layouts/themes/default.html.haml

```diff
@@ -4,6 +4,7 @@
     %title= title
     %meta(name="viewport" content="width=device-width, initial-scale=1.0")
     = stylesheet_link_tag current_website.theme, media: 'all'
+    = javascript_pack_tag "application"
```

Comment:
I wonder if we should have a separate webpack for the front end.

Reply:
@jonsgreen We should look at this when you get back. It would be a good idea to potentially make a separate pack.